code/LeftAndMain.php

```diff
@@ -1740,19 +1740,22 @@ public function CMSVersion()
 
     /**
      * Return the version number of the CMS, ie. '4.2.1'
```

Comment:
```suggestion
     * Return the version number of the CMS in the 'major.minor' format, e.g. '4.2'
```

Reply:
Updated

Reply:
Missed the patch deletion from the original version example @emteknetnz